src/features/Chat/components/ActiveWindow/MessageField.tsx

```diff
@@ -17,36 +20,53 @@ import TextInput from '@/components/TextInput';
 
 type Props = {
   chat: ChatBuddy;
-  sendMessage: (msg: NewMessage) => void;
-  isMessageSendLoading: boolean;
 };
 
-const MessageField = ({ chat, sendMessage, isMessageSendLoading }: Props) => {
+const MessageField = ({ chat }: Props) => {
   const { t } = useTranslation('chat');
   const { isMobile } = useGetLayoutMode();
-  const [text, setText] = useState('');
   const userId = useAppSelector(selectUserId);
+  const [sendMessage] = useSendMessageMutation();
```

Comment:
Oh this returns a tuple, where we can get the isLoading from

```javascript
  const [sendMessage, {isLoading: isMessageSendLoading }] = useSendMessageMutation();
```

no need for local state variable

Reply:
ahaa yes ofcourse, then this is ok!